OPCB/insurgency/common/server/AI/functions.sqf

```diff
@@ -278,10 +278,10 @@ createRoofGun = {
 	_ai moveInGunner _gun;	
 	_grp setFormDir _dir;
 	_grp deleteGroupWhenEmpty true;
+
+	[_gun] call CHAB_fnc_serverGroups;
```

Comment:
Why do you need this? I want to keep insurgency portable and not dependent on OPCB